scripts/dpanon.py

```diff
@@ -44,7 +45,17 @@ def lock_directory(in_dir: Union[Path, str],
     for in_file in Path(in_dir).glob('*'):
         if in_file.is_file():
             out_file = Path(out_dir) / in_file.name
-            module.remove_pii(in_file, out_file)
+            if datatype == 'surveys':
+                pii_table_loc = kwargs.get('pii_table_loc', False)
+                # using the file name without extension as the subject_id input
+                # for the process_and_copy_db function
+                SURVEYS.process_and_copy_db(
+                        pii_table_loc,
+                        in_file.name.split('.')[0],
+                        in_file,
+                        out_file)
+            else:
+                module.remove_pii(in_file, out_file)
```

Comment:
Is this line a placeholder for future `datatype` blocks?

Reply:
`module.remove_pii` is the placeholder for PII scrubbing blocks for each module